src/sentry/tagstore/base.py

```diff
@@ -82,14 +107,27 @@ def get_or_create_tag_value(self, project_id, key, value, **kwargs):
         """
         raise NotImplementedError
 
+    def create_group_tag_key(self, project_id, group_id, key, **kwargs):
+        """
+        >>> create_group_tag_key(1, 2, "key1")
+        """
+        raise NotImplementedError
+
+    def get_or_create_group_tag_key(self, project_id, group_id, key, **kwargs):
+        """
+        >>> get_or_create_group_tag_key(1, 2, "key1")
+        """
+        raise NotImplementedError
+
     def get_tag_key(self, project_id, key, status=TagKeyStatus.VISIBLE):
         """
         >>> get_tag_key(1, "key1")
         """
         raise NotImplementedError
 
-    def get_tag_keys(self, project_id, keys=None, status=TagKeyStatus.VISIBLE):
+    def get_tag_keys(self, project_ids, keys=None, status=TagKeyStatus.VISIBLE):
```

Comment:
Unifying APIs here. Support `list` or `int` for most of them (where it makes sense).